document/CoreFormat.mdk

```diff
@@ -585,38 +589,38 @@ shall either both be present or both be absent.
 that identifies the file that the static analysis tool was instructed to scan.
-This need not be the same as the file where the issue actually occurs. See `issueFile` (&sect;[#location-issueFile]) for more information on this point.
+This need not be the same as the file where the result actually occurs. See `resultFile` (&sect;[#location-resultFile]) for more information on this point.
 
-### `issueFile` property { #location-issueFile }
+### `resultFile` property { #location-resultFile }
 
-A `location` object may contain a property named `issueFile` whose value is a `physicalLocation` object (&sect;[#physicalLocation-info])
-that identifies the file where the static analysis tool detected the issue.
+A `location` object may contain a property named `resultFile` whose value is a `physicalLocation` object (&sect;[#physicalLocation-info])
+that identifies the file where the static analysis tool produced the result.
 
-If the issue actually occurs at the location specified in the `analysisTarget` property,
-then the `issueFile` property shall not be present.
-If the issue occurs in a different file from the one specified in the `analysisTarget` property,
-then the `issueFile` property shall be present.
+If the result actually occurs at the location specified in the `analysisTarget` property,
+then the `resultFile` property shall not be present.
+If the result occurs in a different file from the one specified in the `analysisTarget` property,
+then the `resultFile` property shall be present.
 
 ~ IsoNumberedExample { @IsoNumberedExample=1 }
 Suppose a static analysis tool for C++ source code is instructed to scan the source file _a.cpp_,
 which includes the header file _b.h_,
-and suppose the tool detects an issue in _b.h_.
+and suppose the tool produces a result in _b.h_.
 In this case, the tool should report the `analysisTarget` as _a.cpp_,
-and it should report the `issueFile` as _b.h_.
+and it should report the `resultFile` as _b.h_.
 ~
 
 ~ IsoNumberedExample
-Suppose a static analysis tool for object code detects an issue in the binary file _c.dll_,
+Suppose a static analysis tool for object code produces a result in the binary file _c.dll_,
 and suppose the tool has available symbol information which maps that location within the binary
 to a specific line in a source file _d.cpp_.
 In this case, the tool should report the `analysisTarget` as _c.dll_,
-and it should report the `issueFile` as _d.cpp_.
+and it should report the `resultFile` as _d.cpp_.
 ~
 
 ### `logicalLocation` property { #location-logicalLocation }
 
 A `location` object may contain a property named `logicalLocation` whose value is a `logicalLocation` object (&sect;[#logicalLocation-info])
-which specifies the logical location where the static analysis tool detected the issue.
+which specifies the logical location where the static analysis tool produced the result.
```

Comment:
@michaelcfanning I'm going to leave this one. If a tool doesn't "produce a result" at a logical location, then I don't see how it "detects a result" there. And IMO "logical location" and "physical location" are two equivalent ways of specifying the same thing; if I can "produce a result" at a physical location, I can "produce a result" at the equivalent logical location.